Dockerfile

```diff
@@ -5,8 +5,19 @@ MAINTAINER kramos <markosrendell@gmail.com>
 # Install cloud foundry cli
 RUN apt-get update -y && \
     apt-get install -y curl && \
-    curl -L "https://cli.run.pivotal.io/stable?release=linux64-binary&source=github" | tar xzv -C /usr/local/bin cf && \
-    cf install-plugin autopilot -f -r CF-Community
+    curl -L "https://cli.run.pivotal.io/stable?release=linux64-binary&source=github" | tar xzv -C /usr/local/bin cf
+
+# Install the cloud foundry autopilot plugin
+RUN export LATEST_RELEASE_URL=$(curl -s https://github.com/contraband/autopilot/releases | grep "contraband/autopilot/releases/download" | grep linux | head -n 1 | cut -d '"' -f 2) &&\
```

Comment:
I don't know if you want to  pull `jq` in, but the [github API](https://developer.github.com/v3/repos/releases/#get-the-latest-release) makes this much easier because there's a latest release endpoint. :man_shrugging:

Reply:
would we be worried about hitting any rate limits if we do it that way? @adborden